lib/livebook/session/data.ex

```diff
@@ -57,7 +57,7 @@ defmodule Livebook.Session.Data do
           revision_by_client_pid: %{pid() => cell_revision()},
           evaluation_digest: String.t() | nil,
           evaluation_start_time: float(),
-          last_evaluation_time: float()
+          evaluation_time_ms: float() | nil
```

Comment:
Actually, I think this is `integer() | nil` now as we convert directly to ms without decimal part, and this seems reasonable (otherwise we could just store nanoseconds integer or whatever unit we use, but milliseconds is fine).